src/app/loadout-builder/ModPicker.tsx

```diff
@@ -198,7 +199,7 @@ class ModPicker extends React.Component<Props, State> {
             ? specialtyModSocketHashes.includes(mod.plug.plugCategoryHash)
             : mod.plug.plugCategoryHash === category
         )
-        .map((mod) => ({ mod, category }));
+        .map((mod) => ({ key: uuidv4(), mod, category }));
```

Comment:
You don't really need a whole UUID - just a global counter.

Reply:
Done